pkg/dao/repository_configs.go

```diff
@@ -175,11 +175,11 @@ func (r repositoryConfigDaoImpl) List(
 	}
 
 	if filterData.AvailableForArch != "" {
-		filteredDB = filteredDB.Where("arch = ? OR arch = ''", filterData.AvailableForArch)
+		filteredDB = filteredDB.Where("arch = ? OR arch = '' OR arch = 'any'", filterData.AvailableForArch)
 	}
 	if filterData.AvailableForVersion != "" {
 		filteredDB = filteredDB.
-			Where("? = any (versions) OR array_length(versions, 1) IS NULL", filterData.AvailableForVersion)
+			Where("? = any (versions) OR versions @> ARRAY['any'] OR array_length(versions, 1) IS NULL", filterData.AvailableForVersion)
```

Comment:
hrmm, this is giving me:  ERROR: operator does not exist: character varying[] @> text[] (SQLSTATE 42883)

Reply:
could you just do    `OR 'any' = any (versions)` ?

Reply:
sure